src/api/auth/login/login.module.ts

```diff
@@ -1,15 +1,30 @@
 import { Module } from '@nestjs/common';
 import { PassportModule } from '@nestjs/passport';
+import { JwtModule } from '@nestjs/jwt';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { User } from '../../user/user.entity';
 import { UserService } from '../../user/user.service';
 import { LocalStrategy } from '../strategies/local.strategy';
 import { LoginController } from './login.controller';
 import { LoginService } from './login.service';
+import * as dotenv from 'dotenv';
```

Comment:
Nit: wonder if this can be placed in the root app

Reply:
You did mention this before; I'll make a change now!